org.eclipse.xtext.ide.tests/META-INF/MANIFEST.MF

```diff
@@ -14,8 +14,8 @@ Require-Bundle: org.eclipse.xtext;bundle-version="2.34.0";visibility:=reexport,
  org.eclipse.core.runtime;bundle-version="3.24.100",
  org.eclipse.xtext.ide;bundle-version="2.34.0",
  org.junit;bundle-version="4.13.2",
- org.eclipse.lsp4j;bundle-version="[0.21.1,0.22.0)";resolution:=optional,
- org.eclipse.lsp4j.jsonrpc;bundle-version="[0.21.1,0.22.0)";resolution:=optional,
+ org.eclipse.lsp4j;bundle-version="[0.21.2,0.22.0)";resolution:=optional,
```

Comment:
Should we add version constraints to xtext.testing, too?

Reply:
good question

Reply:
dont know why we havent

Reply:
I'd assume it's an oversight.